test/upload_validation_test.ts

```diff
@@ -2507,6 +2555,32 @@ describe('Pack metadata Validation', () => {
         ]);
       });
 
+      it('column format names are allowed to have funky characters', async () => {
```

Comment:
Should they? I think they're just shown as a label to a user, so... maybe this is intentional?

Reply:
I am also curious. @jonathan-codaio @alexd-codaio

Reply:
Yeah looks like the format name doesn't matter, the formula catalog gets a formula called `Format${formulaName}` where the suffix is the _formula_ that the format references, not the format's name itself. So this seems ok, if confusingly permissive compared to everything else.